apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -928,6 +970,11 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
                 menuItem: t("console:develop.features.applications.edit.sections.advanced.tabName"),
                 render: AdvancedSettingsTabPane
             },
+            {
+                componentId: "shared-access",
+                menuItem: "Shared Access",
```

Comment:
Lets add i18n strings in a followup PR.
```suggestion
                menuItem: "Shared Access",
```